src/lib/Containers/Inquiry.tsx

```diff
@@ -0,0 +1,183 @@
+`
+const ResponseRate = styled(SmallHeadline)`
+  color: ${colors["yellow-bold"]}
+  marginTop: 5
+`
+const ResponseIndicator = styled.View`
+  width: 8
+  height: 8
+  border-radius: 4
+  margin-top: 5
+  margin-right: 5
+  background-color: ${colors["yellow-bold"]}
+`
+const ResponseRateLine = styled.View`
+  flex: 1
+  flex-direction: row
+  align-items: center
+  min-height: 12
+  margin-top: 5
+`
+
+export class Inquiry extends React.Component<RelayProps, any> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      text: this.props.inquiry.partner.contact_message,
+    }
+  }
+
+  dismissModal() {
+    ARSwitchBoard.dismissModalViewController(this)
+  }
+
+  sendInquiry() {
+    fetch(gravityURL + "/api/v1/me/artwork_inquiry_request", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        artwork: this.props.inquiry.id,
+        message: this.state.text,
+      }),
+    })
+    this.dismissModal()
+  }
+
+  render() {
+    const message = this.state.text
+    const partnerResponseRate = "2 DAY RESPONSE TIME"
+    const artwork = this.props.inquiry
+    console.log(this.props)
+    const partnerName = this.props.inquiry.partner.name
+
+    const doneButtonStyles = {
+      backgroundColor: colors["purple-regular"],
+      marginBottom: 0,
+      paddingTop: 12,
+      height: 44,
+    }
+
+    return (
+      <Container>
+        <BottomAlignedButton onPress={this.sendInquiry.bind(this)} bodyStyle={doneButtonStyles} buttonText="SEND">
+          <Header>
+            <HeaderTextContainer>
+              <CancelButton onPress={this.dismissModal.bind(this)}><MetadataText>CANCEL</MetadataText></CancelButton>
+              <TitleView>
+                <SmallHeadline>{partnerName}</SmallHeadline>
+                <ResponseRateLine>
+                  <ResponseIndicator />
+                  <ResponseRate>{partnerResponseRate}</ResponseRate>
+                </ResponseRateLine>
+              </TitleView>
+              <PlaceholderView>CANCEL</PlaceholderView>
+            </HeaderTextContainer>
+          </Header>
+          <Content>
+            <ArtworkPreview artwork={artwork} />
+            <InquiryTextInput
+              value={message}
+              keyboardAppearance={"dark"}
+              multiline={true}
+              autoFocus={true}
+              onEndEditing={() => {
+                this.setState({ active: false, text: null })
+              }}
+              onChangeText={text => this.setState({ text })}
+            />
+          </Content>
+        </BottomAlignedButton>
+      </Container>
+    )
+  }
+}
+
+export default Relay.createContainer(Inquiry, {
+  fragments: {
+    inquiry: () => Relay.QL`
+      fragment on Artwork {
+        id
+        ${ArtworkPreview.getFragment("artwork")}
+        partner {
+          name
+          contact_message
```

Comment:
This is actually [deprecated](https://github.com/artsy/metaphysics/blob/c3f0e0486ab9461b63c1016ebbcc0ce6a0af38c9/schema/partner.js#L54) and the one under `artwork` might be a better fit. It has slightly different text based on the artwork's availability (it's what we use in Force).